css/style.css

```diff
@@ -4,7 +4,114 @@
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    margin: 0;
+    padding: 0;
+    border: 1px solid white
 }
+*{transition:all 0.3s ease-in-out;}
+*{transition:all 0.3s ease-in-out;}
+
+.container{
+    clear: both;
+    overflow: auto;
+}
+
+nav{float:right;}
+
+.logo img{
+    position: relative;
+    float: left;
+    width:  50px;
+    height: 50px;
+}
+
+ul li{
+  display: inline-block;
+   padding:10px;
+}
+
+ul li:hover{
+  color:orange;
+  
+}
+
+.bckimg{
+    background-image: url(/img/first-background.jpg);
```

Comment:
Did this path work for you? It doesn't for me. I've had to add 2 dots for it to work: `../img/first-background.jpg`